### ISSUE TITLE
Custom datasets

### ISSUE THREAD

**Issue by bhack** [yoxu515/aot-benchmark#30]
I see in the readme an empty section to train/fine-tune on custom dataset.

Do you have any news to share on this topic?

**Comment by z-x-yang** [comment]
I'll work on this when I have some free time.

By the way, training/fine-tuning on a custom dataset is not hard to implement. A simple way for you to reach this on the current code is that you can replace the YouTube-VOS/DAVIS dataset with your custom dataset. 

The dataset format is simple, that is a folder for images and another for masks. Images and masks of different sequences are placed in different sub-folders named by unique sequence names. Please check the YouTube-VOS/DAVIS dataset files for more details, which will be simple for you to understand.

**Comment by bhack** [comment]
Yes, probably in the meantime, the interesting part  is about few line in the markdown on a suggested protocol on what part of the network to fine tune and what to freeze if experimenting with the model zoo.

**Comment by z-x-yang** [comment]
When pretraining and finetuning, we only freeze the first two stages and the BN statistics of the backbone as reported in our paper.

**Comment by bhack** [comment]
Any specific hint about the parameters if I would try to test a training job with Davis fullres or more in general at higher resolution to extract more details?

**Comment by z-x-yang** [comment]
You could try to set --max_resolution to a resolution you want to extract more details.
